src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/__tests__/ExposureSubmissionCoordinatorModelTests.swift

```diff
@@ -63,6 +65,7 @@ class ExposureSubmissionCoordinatorModelTests: XCTestCase {
 		XCTAssertFalse(model.exposureSubmissionServiceHasRegistrationToken)
 	}
 
+	// MARK: -
```

Comment:
Why are empty marks used here?
The description of the test method should be sufficient here.